src/MoBi.Presentation/Presenter/SelectNeighborPathPresenter.cs

```diff
@@ -43,11 +44,13 @@ private void onSelectedContainerPathChanged(IEntity entity, ObjectPath container
          if (container.Mode != ContainerMode.Physical) 
             return;
 
-         _selectedPathDTO.Path = containerObjectPath.PathAsString;
+         var parentPath = container.RootContainer.ParentPath?.Clone<ObjectPath>() ?? new ObjectPath();
```

Comment:
@Yuri05 I am changing thius so that if we select a container with a root container with a parent path, we add the parent path as well.